app/services/ping.rb

```diff
@@ -4,11 +4,14 @@ class Ping
 
   class << self
     def ping
-      {
+      response = {
         'foreman': {
           database: ping_database,
         },
       }.merge(plugins_ping)
+
+      response['foreman']['cache'] = ping_cache if redis_cache_store?
```

Comment:
```suggestion
      response[:foreman]['cache'] = ping_cache if redis_cache_store?
```

otherwise I get:
```
2023-11-14T09:10:14 [I|app|e8717c9e] Started GET "/api/ping" for 192.168.122.1 at 2023-11-14 09:10:14 +0000
2023-11-14T09:10:14 [I|app|e8717c9e] Processing by Api::V2::PingController#ping as JSON
2023-11-14T09:10:14 [I|app|e8717c9e]   Parameters: {"apiv"=>"v2"}
2023-11-14T09:10:14 [W|app|e8717c9e] Action failed
2023-11-14T09:10:14 [I|app|e8717c9e] Backtrace for 'Action failed' error (RuntimeError): {:foreman=>{:database=>{:active=>true, :duration_ms=>"0"}}}
 e8717c9e | /usr/share/foreman/app/services/ping.rb:13:in `ping'
 e8717c9e | /usr/share/foreman/app/controllers/api/v2/ping_controller.rb:14:in `ping'
```

And if we're already changing things, this makes it more consistent:
```suggestion
      response[:foreman][:cache] = ping_cache if redis_cache_store?
```